samples/sensor/lsm303dlhc/README.rst

```diff
@@ -0,0 +1,51 @@
+.. _lsm303dlhc:
+
+LSM303DLHC: Magnetometer and Accelerometer data Monitor
+#######################################################
+
+Overview
+********
+This sample application periodically reads magnetometer and accelerometer data
+from the LSM303DLHC, ST MEMS system-in-package featuring a 3D digital linear
```

Comment:
since you repeat the details below, this sentence reads better without the details, so how about:

    This sample application periodically reads magnetometer and accelerometer data
    from the LSM303DLHC eCompass module's sensors, and displays the sensor data
    on the console.